changelog/unreleased/pull-3167

```diff
@@ -4,6 +4,6 @@ The `--last` option allowed limiting the output of the `snapshots`
 command to the latest snapshot for each host. The new `--latest n`
 option allows limiting the output to the latest `n` snapshots.
 
-This change deprecate `--last` in favour of `--latest 1`
+This change deprecates the option `--last` in favour of `--latest 1`.
 
 https://github.com/restic/restic/pull/3167
```

Comment:
Couldn't add a comment to the top-most line. Suggest changing heading to "Enhancement: Allow specifying limit of `snapshots` list".